### ISSUE TITLE
Warrior Prot Spec & Talents Seem to be Completely Bugged

### ISSUE THREAD

**Issue by scairen** [dalaranwow/dalaran-wow#4891]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Warrior Prot Talents seem to be completely bugged as of Oct 3, and threat is disappearing with no explanation. Multiple abilities are not being properly modified by talents. I had a guildy jump on my warrior and test it out, and he experienced the same issues. I have a 5,500gc and I'm being creamed by bosses that should be push-overs for my gearscore and guild healers. This indicates to me that something with the scripting is messed up. 

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Enter Defensive stance, activate Prot spec, and tank something with 2-3 dps going at typical raid rate.
2. 
3. 

**Include proofs for this behaviour**

Short of me posting a video of my 10-man ICC raid, there not a lot I can put here to prove that my Prot spec talents aren't working. 

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Crispybuilt** [comment]
I'll get on to this issue as soon as I come back home, though I highly doubt something like that happened overnight.

**Comment by Byssarod** [comment]
Can confirm. I tested it and seems Thunder Clap, Heroic Throw and Cleave threat multiply is not counting.

**Comment by ashido96** [comment]
If you receive a blessing of protection while in prot all your bonus threat goes by by, be warry

**Comment by TeridanDW** [comment]
Thank you for the comment @ashido96, regarding the issue I think @Rushor fixed it in #4365.
Will reopen this report if the problem persists after #4365 is on live.
